system/lib/libc/emscripten_memset.c

```diff
@@ -1,8 +1,9 @@
-// XXX EMSCRIPTEN ASAN: build an uninstrumented version of memset
-#if defined(__EMSCRIPTEN__) && defined(__has_feature)
-#if __has_feature(address_sanitizer)
-#define memset __attribute__((no_sanitize("address"))) emscripten_builtin_memset
```

Comment:
What happened to `emscripten_bultin_memset`?

Reply:
Yeah, most of the `emscripten_builtin_*` functions are used by the sanitizers since we don't have real interposition.